chapters/ch33.md

```diff
@@ -1,28 +1,28 @@
 
 
 > Further, there are several reasons building an ML system is easier if you are trying to do a task that people can do well:
 
- Hơn nữa, việc xây dựng một hệ thống ML dễ dàng hơn nếu bạn đang cố gắng thực hiện một tác vụ mà mọi người có thể làm tốt dựa theo một số lí do sau:
+ Hơn nữa, có rất nhiều lý do khiến việc xây dựng một hệ thống ML dễ dàng hơn nếu bạn đang giải quyết một tác vụ con người có thể làm tốt:
 
 
 > 1. **Ease of obtaining data from human labelers**.​ For example, since people recognize cat images well, it is straightforward for people to provide high accuracy labels for your learning algorithm.
 
- 1. **Dữ liệu được lấy dễ dàng từ người dán nhãn**. ​ Ví dụ, con người nhận diện hình ảnh mèo tốt nên việc mọi người cung cấp nhãn có độ chính xác cao cho thuật toán học tập là điều đơn giản.
+ 1. **Dữ liệu được gán nhãn bởi con người một cách dễ dàng**. ​ Ví dụ, con người nhận diện hình ảnh mèo tốt nên việc mọi người cung cấp nhãn có độ chính xác cao cho thuật toán học tập là điều đơn giản.
 
 > 2. **Error analysis can draw on human intuition**.​ Suppose a speech recognition algorithm is doing worse than human-level recognition. Say it incorrectly transcribes an audio clip as “This recipe calls for a ​*pear​* of apples,” mistaking “pair” for “pear.” You can draw on human intuition and try to understand what information a person uses to get the correct transcription, and use this knowledge to modify the learning algorithm.
 
- 2. **Phân tích lỗi có thể dựa vào trực giác của con người**. ​ Giả sử rằng 1 thuật toán nhận dạng giọng nói làm tệ hơn so với con người. Giả dụ nó ghi nhầm một đoạn âm thanh thành “công thức này cần một cục táo” thay vì “cặp táo.” Bạn có thể dựa vào trực giác và cố gắng hiểu thông tin nào đã được sử dụng để thu được bản ghi chuẩn, và dùng thông tin này để điều chỉnh thuật toán.
+ 2. **Phân tích lỗi có thể dựa vào trực giác của con người**. ​ Giả sử rằng một thuật toán nhận dạng giọng nói làm tệ hơn so với con người. Giả dụ nó ghi nhầm một đoạn âm thanh thành “công thức này cần một cục táo” thay vì “cặp táo.” Bạn có thể dựa vào trực giác và cố gắng hiểu thông tin nào một người sử dụng để thu được bản ghi chuẩn, và dùng thông tin này để điều chỉnh thuật toán.
 
 > 3. **Use human-level performance to estimate the optimal error rate and also set a “desired error rate.”​** Suppose your algorithm achieves 10% error on a task, but a person achieves 2% error. Then we know that the optimal error rate is 2% or lower and the avoidable bias is at least 8%. Thus, you should try bias-reducing techniques.
 
- 3. **Sử dụng hiệu quả cấp con người để ước tính tỷ lệ lỗi tối ưu cũng như đặt ra "tỷ lệ lỗi mong muốn."​** Giả sử thuật toán của bạn trả về 10% lỗi trong một tác vụ, nhưng một người chỉ lỗi 2%. Dựa vào đó, chúng ta biết rằng tỷ lệ lỗi tối ưu là 2% hoặc thấp hơn và giảm độ chệch có thể tránh được ít nhất là 8%. Vì vậy, bạn nên thử các kỹ thuật giảm độ chệch.
```

Comment:
Ở cụm "the avoidable bias is at least 8%" dịch giữ nguyên bản gốc là "độ chệch có thể tránh ít nhất là 8%", đây là một cấu trúc ngữ pháp "S + to be + N" nên không nên dịch sang "S + V".